<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo1.png</normaloff>logo1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>421</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solidrgb(255, 85, 127);
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0.06, y2:0.131227, stop:0.373134 rgba(249, 148, 207, 255), stop:1 rgba(255, 255, 255, 255))</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{
	border-radius:20px;
	border:2px solid rgb(255, 0, 127);
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.713, fx:0.5, fy:0.5, stop:0.159204 rgba(228, 76, 147, 255), stop:0.890547 rgba(255, 255, 255, 255))
}

QPushButton:hover{
	
	background-color: rgb(255, 255, 255);

}



</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-bottom:2px solid rgb(255, 0, 127);
padding-bottom:7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxUsuario">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(255, 0, 127)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::NoInsert</enum>
    </property>
    <item>
     <property name="text">
      <string>[Usuario]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administrador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cajero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almacen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Venta</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rage Italic</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Bienvenido&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblError2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
